README.md

```diff
@@ -1,13 +1,12 @@
 ![Legato logo](http://legato.io/resources/img/legato_logo.png)
- 
-==================
+
+---
 
 ![Build Status](https://travis-ci.org/legatoproject/legato-af.svg)
 
-Legato Open Source Project started out as an initiative by Sierra Wireless Inc. to provide an open, secure and easy to use Application Framework to lower the barrier for developers, who are not familiar with embedded programming, to participate in the exponential growth of "Internet of Things".
-Visit [legato.io](legato.io) to learn more or visit the [Legato forum](forum.legato.io).
+Legato Open Source Project started out as an initiative by Sierra Wireless Inc. to provide an open, secure and easy to use Application Framework to lower the barrier for developers, who are not familiar with embedded programming, to participate in the exponential growth of "Internet of Things". Visit [legato.io](legato.io) to learn more or visit the [Legato forum](forum.legato.io).
```

Comment:
I think this is a run-on sentence.  *disclaimer*: not an English major

Here's my attempt at splitting it up:

> Legato Open Source Project is an initiative of Sierra Wireless Inc. which provides an open, secure and easy to use  Application Framework for embedded devices.  The project enables developers who are not experienced in traditional embedded programming to participate in the exponential growth of the "Internet of Things".